### PR TITLE
Updated initiative content

### PR DESCRIPTION
**Pull request by OphyBoamah**
Resolves: https://github.com/thegooddocsproject/website-hugo/issues/168

### CODE REVIEW

File: content/en/_index.md [hunk 1]
```diff
@@ -47,18 +47,9 @@ Our key target audiences are:
...
-* How-to for style guides
-* Base template (template for templates)
-
-And possibly:
-* Docs audit framework
-* Implementing a style guide in tools
-* Integrate doc tool chains
-* Doc format conversion tools
+## 2022 Initiatives
+For a list of our current projects and initiatives, see our [2022 release goals](https://github.com/thegooddocsproject/governance/wiki/Write-The-Docs-Portland-2022-Release)
```
Comment (barbaricyawps) [inline content/en/_index.md:51]: Hi, Ophy. Great work on this PR! I have only a minor request: can you add a period at the end of this sentence?

**Review by barbaricyawps — CHANGES_REQUESTED** [review]
Looks good! And congratulations on getting close to finishing your Git homework assignment!

 However, I'm not sure why your PR is pulling in some other changes. Did you make sure that you pulled upstream changes before creating your branch? Also, you might need to rebase because it's possible some other changes were made to the website from the time you created the branch to the time that you opened a pull request.

Can you run these commands inside your `index-content` branch:

`git fetch upstream`
`git pull upstream main`

Then push those changes up to this PR? If that doesn't work, we might have to do a Git rebase.

**Review by bwklein — CHANGES_REQUESTED** [review]
Please add `.idea` to the root level .gitignore file.  We don't want local IDE config files in the repo.

**Review by bwklein — APPROVED** [review]
Thank you, looks great!  I removed the IDE files from the PR too.

File: content/en/_index.md (hunk 1 above)
Comment (bwklein) [reply content/en/_index.md]: This was done in the last update.

**Comment by bwklein** [comment]
@barbaricyawps should be good to go now.

**Review by barbaricyawps — APPROVED** [review]
Everything looks great to me too! Congrats, @OphyBoamah ! I'll merge this in!

**Pull request merged**